opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md

```diff
@@ -0,0 +1,48 @@
+# Java Agent
+
+This example shows the use of the [Splunk distribution of OpenTelemetry Java instrumentation project](https://github.com/signalfx/splunk-otel-java).  
+The Java agent performs bytecode manipulation of the target application using the [JVM
+Instrumentation](https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/package-summary.html)
+interface and enables tracing of many popular libraries and frameworks without
+modification of the target application's source code.
+
+Here are some environment variables you might need to set to configure the Java
+agent:
+```
+$ # Exporter configuration - default is OTEL exporter
+$ # Eg. set ZIPKIN exporter - compatible with SignalFX SmartAgent
+$ export OTEL_EXPORTER=zipkin
+
+$ # Exporter endpoint URL - each exporter has own, specific property name and the default value  
+$ # Eg. set ZIPKIN exporter URL - default is http://localhost:9411/api/v2/spans
```

Comment:
This is no longer true - see https://github.com/signalfx/splunk-otel-java/blob/main/agent/src/main/java/com/splunk/opentelemetry/SplunkAgent.java#L34

Reply:
It's hard do judge if we should refer to general agent or Splunk (temporary?) distribution. I decided to stick with general behaviour, but can change that of course. @iNikem what do you think?

Reply:
If this example references our (Splunk) distribution, then it should be consistent with its behaviour. Including defaults.

Reply:
Changed to "splunk default" values.